packages/fxa-shared/test/db/mysql.ts

```diff
@@ -143,4 +159,23 @@ describe('mysql', function () {
       dbOpts.connectionLimit * 2
     );
   });
+
+  it('closes connections after idling', async () => {
+    // pass a lower idle limit config value
+    mysql = new MySqlStoreTest({ ...dbOpts, idleLimitMs: 10 });
+    const conn1 = await mysql.getConnection();
+    const conn2 = await mysql.getConnection();
+    assert.equal((conn1 as Connection).state, 'connected');
+    assert.equal((conn2 as Connection).state, 'connected');
```

Comment:
It seems like this can vary between authenticated and connected. Maybe it is not neccessary to check here?

Reply:
Good point.  I can assert that it is not 'disconnected', which is the initial state.